__tests__/components/AppLayout/SidebarItems/FeedbackNavigationItem.test.tsx

```
@@ -36,189 +42,209 @@ jest.mock("@ui5/webcomponents-react", () => ({
 
-describe("FeedbackNavigationItem", () => {
+describe('FeedbackNavigationItem', () => {
   const mockOnSubmitFeedback = jest.fn();
 
   beforeEach(() => {
     jest.clearAllMocks();
-    mockCreateFeedback.mockResolvedValue({ id: "123", message: "Success" } as any);
+    mockCreateFeedback.mockResolvedValue({
+      id: '123',
+      message: 'Success',
+    } as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)